packages/admin-ui-docs/docs/guidelines/components/switch.mdx

```diff
@@ -0,0 +1,284 @@
+  const [checked, setChecked] = React.useState(false)
+
+  return (
+    <Switch
+      checked={checked}
+      onChange={() => setChecked(!checked)}
+      label="Label"
+    />
+  )
+}
+```
+
+### useSwitchState
+
+For convenience, we also provide a hook that already implements the state logic for you. It can be very handy if you have a group of switches and want to handle the states of each one. You should pass the hook return to the `state` property and also define a `value` for each `Switch`.
+
+```jsx live
+function Example() {
+  const switchState = useSwitchState({ state: false })
+
+  return <Switch state={switchState} label="Label" />
+}
+```
+
+## Props
+
+| Name      | Type             | Description                                                                                                                                   | Required | Default |
+| --------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- |
+| label     | `ReactNode`      | The component's label                                                                                                                         | ✅       | -       |
+| helpText  | `ReactNode`      | Help text                                                                                                                                     | 🚫       | -       |
+| errorText | `ReactNode`      | Error text                                                                                                                                    | 🚫       | -       |
+| error     | `boolean`        | Whether the switch contains an error or not                                                                                                   | 🚫       | -       |
+| checked   | `boolean`        | Switch's checked state. If present, it is used instead of state                                                                               | 🚫       | -       |
+| disabled  | `boolean`        | Defines if the Switch is disabled or not                                                                                                      | 🚫       | -       |
+| value     | `string, number` | Switch's value is going to be used when multiple switches share the same state. Checking a switch with a value will add it to the state array | 🚫       | -       |
+| state     | `SwtichState`    | Return of `useSwitchState` hook. You can also provide these props from your own state logic.                                                  | 🚫       | -       |
+| csx       | `StyleProp`      | Defines component styles                                                                                                                      | 🚫       | `{}`    |
+
+> The Switch component also supports all HTML props from the `<input />` HTML tag.
+
+</Panel>
+<Panel id="designReference">
+
+## Do’s
```

Comment:
Why not using the tabs in this doc?

Reply:
I didn't get it. Maybe I'm not holding all the context. Do you mean these tabs?:

https://user-images.githubusercontent.com/33183880/167321383-fb065407-ef5a-404e-b980-4302ab000a27.mov